src/resend.spec.ts

```diff
@@ -160,4 +167,38 @@ describe('Resend', () => {
       }
     `);
   });
+
+  it('can send an email with headers', async () => {
+    const payload: CreateEmailOptions = {
+      from: 'admin@resend.com',
+      headers: {
+        'X-Entity-Ref-ID': '123',
+      },
+      subject: 'Hello World',
+      text: 'Hello world',
+      to: 'bu@resend.com',
+    };
+    mock.onPost('/emails', payload).replyOnce(200, {
+      id: '1234',
+      from: 'admin@resend.com',
+      to: 'bu@resend.com',
+      headers: {
+        'X-Entity-Ref-ID': '123',
+      },
+      created_at: '123',
+    });
+
+    const data = await resend.emails.send(payload);
+    expect(data).toMatchInlineSnapshot(`
```

Comment:
CreateEmailResponse, only returns `id`

Reply:
Thanks for catching that. I'll get that updated ASAP.